contracts/interfaces/IICS20Transfer.sol

```diff
@@ -51,6 +56,14 @@ interface IICS20Transfer {
         external
         returns (uint64 sequence);
 
+    /// @notice Inserts a custom ERC20 contract for a given IBC denom
+    /// @dev This must be called prior to the first transfer of the token so that the is no existing entry for the
+    /// denom.
+    /// @dev This function requires the `ERC20_CUSTOMIZER_ROLE`
+    /// @param denom The IBC denom
+    /// @param token The address of the custom ERC20 contract
+    function insertCustomERC20(string calldata denom, address token) external;
```

Comment:
Since this is referring to what we everywhere else call ibcERC20, I think it could be slightly clearer if this was just `setCustomIBCERC20`

Reply:
Fair enough. `set` feels like it can override an existing entry whereas insert feels less intrusive. I can use `set`. Wdyt?

Reply:
There are def times where a `set` doesn't allow setting if already existing, so I don't think that would be a problem really. I don't mind terribly, but `set` feels a little more natural for me (but that is subjective ofc :))

Reply:
done.